man/pstore.conf.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version='1.0'?>
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
+<!-- SPDX-License-Identifier: LGPL-2.1+ -->
+
+<refentry id="pstore.conf" conditional="ENABLE_PSTORE"
+          xmlns:xi="http://www.w3.org/2001/XInclude">
+  <refentryinfo>
+    <title>pstore.conf</title>
+    <productname>systemd</productname>
+  </refentryinfo>
+
+  <refmeta>
+    <refentrytitle>pstore.conf</refentrytitle>
+    <manvolnum>5</manvolnum>
+  </refmeta>
+
+  <refnamediv>
+    <refname>pstore.conf</refname>
+    <refpurpose>PStore configuration file</refpurpose>
+  </refnamediv>
+
+  <refsynopsisdiv>
+    <para>
+    <filename>/etc/systemd/pstore.conf</filename>
+    <filename>/etc/systemd/pstore.conf.d/*</filename>
+    </para>
+  </refsynopsisdiv>
+
+  <refsect1>
+    <title>Description</title>
+
+    <para>This file configures the behavior of
+    <citerefentry><refentrytitle>systemd-pstore</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
+    a handler for archiving the contents of the persistent storage filesystem, pstore.
```

Comment:
Maybe add
`<ulink url="https://www.kernel.org/doc/Documentation/ABI/testing/pstore">pstore</ulink>`?

Reply:
I've added the link.

Wrt/ the base comment "that only dmesg is interesting and other files don't need to be processed", I would not quite characterize my stance as such. More specifically, at this point in time I only understand handling the dmesg types; all those other types (console, pmsg, frace, mce, and the various ppc) I simply have no experience with them. Said differently, I don't know how to process/handle them.

For coredumps, there is an option to store them outside the journal, as well as inside the journal. I'd suggest these dmesg (and likely most of the pstore types) are files that would be consumed [post-mortem] similarly to a coredump. I'm suggesting let's keep the ability to write the reconstructed dmesg as a regular file, please.